### PR TITLE
[15.0][mig]-product_manufacturer_partner

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**

### CODE REVIEW

File: product_manufacturer_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+You can access the manufacturer list via a new menu entry 'Manufacturers' on
+'Purchase'.
+
+Credits
+=======
+
+Images
+------
+
+* ForgeFlow S.L. `Icon <http://www.eficent.com/wordpress/wp-content/uploads/2012/09/eficent_logo.png>`_.
```
Comment (AaronHForgeFlow) [inline product_manufacturer_partner/README.rst]: same comment than in previous version

File: product_manufacturer_partner/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Copyright 2015-17 ForgeFlow S.L.
```
Comment (AaronHForgeFlow) [inline product_manufacturer_partner/__init__.py]: remove this please

File: product_manufacturer_partner/README.rst (hunk 1 above)
Comment (JuanyDForgeflow) [reply product_manufacturer_partner/README.rst]: @AaronHForgeFlow Done

File: product_manufacturer_partner/__init__.py (hunk 2 above)
Comment (JuanyDForgeflow) [reply product_manufacturer_partner/__init__.py]: @AaronHForgeFlow Done

**Pull request merged**